akka-http-core/src/main/scala/akka/http/javadsl/settings/SettingsCompanion.scala

```diff
@@ -10,7 +10,7 @@ trait SettingsCompanion[T] {
    *
    * Java API
    */
-  final def create(system: ActorSystem): T = create(system.settings.config)
+  def create(system: ActorSystem): T
```

Comment:
Yes, the CI failed due to mima complaining about breaking binary compatibility like you said. I'll try to put it back the way it was and remove the `final` modifier.

Do you still want the `/** INTERNAL API */` though? At least the comment will convey the intent that it's not meant to be extended by anyone else but us.